arrow/arrow.py

```diff
@@ -641,14 +642,14 @@ def format(self, fmt='YYYY-MM-DD HH:mm:ssZZ', locale='en_us'):
         return formatter.DateTimeFormatter(locale).format(self._datetime, fmt)
 
 
-    def humanize(self, other=None, locale='en_us', only_distance=False):
+    def humanize(self, other=None, locale='en_us', only_distance=False, granularity='auto'):
         ''' Returns a localized, humanized representation of a relative difference in time.
 
         :param other: (optional) an :class:`Arrow <arrow.arrow.Arrow>` or ``datetime`` object.
             Defaults to now in the current :class:`Arrow <arrow.arrow.Arrow>` object's timezone.
         :param locale: (optional) a ``str`` specifying a locale.  Defaults to 'en_us'.
         :param only_distance: (optional) returns only time difference eg: "11 seconds" without "in" or "ago" part.
-
+        :param granularity: (optinal) defines the precision of the output. Set it to strings 'second', 'minute', 'hour', 'day', 'month' or 'year'.
```

Comment:
typo: optinal

Also, it'd be good if you could include some usage examples in the PR comments.